site/content/docs/user/quick-start.md

```diff
@@ -218,26 +218,34 @@ context name `kind` and delete that cluster.
 
 ## Loading an Image Into Your Cluster
 
-You can load one or more images into your kind cluster:
+Docker images can be loaded into your cluster nodes with:
 
-```bash
-kind load docker-image my-app:latest
-```
-
-```bash
-kind load docker-image my-app:latest my-db:latest my-cache:latest
-```
+`kind load docker-image my-custom-image-0 my-custom-image-1`
 
-Note: If using a named cluster you will need to specify the name of the cluster:
+> **Note**: If using a named cluster you will need to specify the name of the
+> cluster you wish to load the images into:
+> `kind load docker-image my-custom-image-0 my-custom-image-1 --name kind-2`
 
-```bash
-kind load docker-image my-app:latest --name test-cluster
-```
 
 Additionally, image archives can be loaded with:
 `kind load image-archive /my-image-archive.tar`
 
+
```

Comment:
Same here. If you use a markdown linter, they typically only want one blank line between elements.